tests/zkevm/test_worst_compute.py

```diff
@@ -1295,15 +1295,23 @@ def test_worst_blobhash(
     """Test running a block with as many BLOBHASH instructions as possible."""
     env = Environment()
     max_code_size = fork.max_code_size()
+    max_stack_height = fork.max_stack_height()
 
-    code_prefix = Op.PUSH1(blob_index) + Op.JUMPDEST
-    code_suffix = Op.JUMP(len(code_prefix) - 1)
-    loop_iter = Op.POP(Op.BLOBHASH(Op.DUP1))
-    code_body_len = (max_code_size - len(code_prefix) - len(code_suffix)) // len(loop_iter)
-    code_body = loop_iter * code_body_len
-    code = code_prefix + code_body + code_suffix
+    # Contract that contains a collection of BLOBHASH instructions.
+    opcode_sequence = Op.BLOBHASH(blob_index) * max_stack_height
+    assert len(opcode_sequence) <= max_code_size
+
+    target_contract_address = pre.deploy_contract(code=opcode_sequence)
+
+    # Contract that contains a loop of STATICCALLs to the target contract.
+    calldata = Bytecode()
+    attack_block = Op.POP(Op.STATICCALL(Op.GAS, target_contract_address, 0, 0, 0, 0))
```

Comment:
Nit if you're interested, I think you can do: `Op.STATICCALL(address = target_contract_address)` and you'll have the same effect.